tools/rapids-install-sccache

```diff
@@ -12,36 +12,15 @@ if test "$SCCACHE_VERSION" = rapids; then
     SCCACHE_REPO="rapidsai/sccache"
 fi
 
-ATTEMPTS=0
-while [ "${SCCACHE_VERSION:-latest}" = latest ]; do
-    SCCACHE_VERSION="$(wget --no-hsts -q -O- "https://api.github.com/repos/${SCCACHE_REPO}/releases/latest" | jq -r '.tag_name | sub("^v"; "")')"
-    if test -n "${SCCACHE_VERSION:+x}"; then
-        break
-    elif test "$((ATTEMPTS++))" -lt 10; then
-        echo "(!) Failed to load latest ${SCCACHE_REPO} version. Retrying (${ATTEMPTS}/10)..." >&2
-        SCCACHE_VERSION="latest"
-        sleep 10
-    else
-        echo "(!) Failed to retrieve latest sccache version after 10 attempts." >&2
-        exit 1
-    fi
-done
+if [ "${SCCACHE_VERSION:-latest}" = latest ]; then
```

Comment:
```suggestion
rapids-prompt-local-github-auth

if [ "${SCCACHE_VERSION:-latest}" = latest ]; then
```

For people reproducing CI locally, could you also add a `rapids-prompt-local-github-auth` around here?

We use that in the `download-*` tools, like this:

https://github.com/rapidsai/gha-tools/blob/fe3bcd8e365125a1578265653d16a10696710719/tools/rapids-github-run-id#L24